### PR TITLE
Add google maven central mirror

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// While maven is throttling travis builds
+ThisBuild / resolvers := ("Google Maven Central Mirror".at(
```
Comment (patriknw) [inline build.sbt]: I haven't got used to this sbt syntax yet, but shouldn't this be `+=`?

Comment (patriknw) [reply build.sbt]: ah, it's at the end -- confusing

Comment (chbatey) [reply build.sbt]: I think that would put it at the end, i wanted it at the start

Comment (patriknw) [reply build.sbt]: I see, ok

**Comment by chbatey** [comment]
Will only merge this is there are more problems

**Pull request closed without merging**